scripts/docker/README.md

```diff
@@ -0,0 +1,76 @@
+# Dockerized components
+
+The dockerfiles in this folder are used to build images for each of the components:
+
+- `node`
+- `bootstrapper`
+- `node-collector`
+
+All the components are compiled (in both release and debug) in a monolithic image from the dockerfile `universal.Dockerfile`.
+The other dockerfiles just extract the individual binaries from this image, install dependencies, and declare exposed ports.
+
+To run the node, the genesis file `genesis.dat` needs to be placed at the data path when the node starts.
+One way to do this is to inject the file as a bind mount as shown in the example below.
+
+## Jenkins Pipelines
+
+The node-related binaries are built by the pipeline `master.Jenkinsfile` in the `jenkinsfiles` top-level folder.
+Genesis images are built with `master-genesis.Jenkinsfile`.
+
+## Docker Compose
+
+The following example shows a (reasonably) minimal setup of a node and an accompanying collector:
+
+```yaml
+version: '3'
+services:
+  node:
+    container_name: node
+    image: ${NODE_IMAGE}
+    networks:
+    - concordium
+    environment:
+    - CONCORDIUM_NODE_CONNECTION_BOOTSTRAP_NODES=bootstrap.${DOMAIN}:8888
+    - CONCORDIUM_NODE_DATA_DIR=/mnt/data
+    - CONCORDIUM_NODE_CONFIG_DIR=/mnt/config
+    - CONCORDIUM_NODE_CONSENSUS_GENESIS_DATA_FILE=/mnt/genesis.dat
+    - CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE=/mnt/baker-credentials.json
+    - CONCORDIUM_NODE_PROMETHEUS_SERVER=1
+    - CONCORDIUM_NODE_RPC_SERVER_ADDR=0.0.0.0
+    ports:
+    - "8888:8888"
```

Comment:
Does this mean that 8888 would be exposed to the host?
If so, why does it need to be? Aren't the nodes communicating on the internal concordium network?

Reply:
This example is for a node that joins an existing network, not a cluster running on an internal docker network. The port needs to be exposed to allow incoming connections.